components/google_calendar/actions/quick-add-event/quick-add-event.js

```diff
@@ -0,0 +1,30 @@
+const googleCalendar = require("../../google_calendar.app");
+
+module.exports = {
+  key: "google_calendar_quick_add_event",
+  name: "Add Quick Event",
+  description: "Create an event to the Google Calendar.",
+  version: "0.0.8",
```

Comment:
If this is a brand new action I recommend setting the version to otherwise omit this comment
```suggestion
  version: "0.0.1",
```